tough/src/cache.rs

```diff
@@ -42,7 +42,7 @@ impl<'a, T: Transport> Repository<'a, T> {
                 self.cache_target(&targets_outdir, target_name.as_ref())?;
             }
         } else {
-            let targets = &self.targets.signed.targets_map();
+            let targets = &self.targets.signed.as_ref().targets_map();
```

Comment:
I'm tempted to suggest `impl<T> Deref for Verbatim<T>` (and `DerefMut`) so that we don't need to `as_ref()`/`as_mut()` everywhere, including in downstream crates. However, I worry about how that works if the inner type has a `inner` or `extra` member or method.